xml/deploy_cephadm.xml

```diff
@@ -990,28 +990,26 @@ o- minions ................................................. [Minions: 5]
     <para>
      To deploy new MONs, run:
     </para>
-<screen>&prompt.cephuser;ceph orch apply mon <replaceable>SHORT_MON_HOST_NAME</replaceable></screen>
+<screen>&prompt.cephuser;ceph orch apply mon <replaceable>SHORT_MON_HOST_NAMES</replaceable></screen>
     <para>
      To deploy new MGRs, run:
     </para>
-<screen>&prompt.cephuser;ceph orch apply mgr <replaceable>SHORT_MGR_HOST_NAME</replaceable></screen>
+<screen>&prompt.cephuser;ceph orch apply mgr <replaceable>SHORT_MGR_HOST_NAMES</replaceable></screen>
     <para>
      For example:
     </para>
 <screen>
-&prompt.cephuser;ceph orch apply mon ses-min1
-&prompt.cephuser;ceph orch apply mgr ses-min1
+&prompt.cephuser;ceph orch apply mon ses-min1,ses-min2,ses-min3
+&prompt.cephuser;ceph orch apply mgr ses-min1,ses-min2,ses-min3
 </screen>
     <tip>
      <para>
       If MONs or MGRs are <emphasis>not</emphasis> on the same subnet, you need
       to append the subnet address. For example:
      </para>
 <screen>
-&prompt.cephuser;ceph orch apply mon ses-min1:10.1.2.0/24
-&prompt.cephuser;ceph orch apply mon ses-min2:10.1.5.0/24
-&prompt.cephuser;ceph orch apply mgr ses-min1:10.1.2.0/24
-&prompt.cephuser;ceph orch apply mgr ses-min2:10.1.5.0/24
+&prompt.cephuser;ceph orch apply mon ses-min1:10.1.2.0/24,ses-min2:10.1.5.0/24
+&prompt.cephuser;ceph orch apply mgr ses-min1:10.1.2.0/24,ses-min2:10.1.5.0/24
```

Comment:
Unless I am misunderstanding, is `ses-min3` missing here?

Reply:
maybe a good idea, i'll add 3rd monitor to each command shortly...

Reply:
although it's not necessary....it looks better